<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14623" started-at="2018-06-03T04:53:17Z" finished-at="2018-06-03T04:53:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="14623" started-at="2018-06-03T04:53:17Z" finished-at="2018-06-03T04:53:32Z">
      <class name="home.rainford.testexecution">
        <test-method status="PASS" signature="verify_launch()[pri:0, instance:home.rainford.testexecution@604ed9f0]" name="verify_launch" is-config="true" duration-ms="8164" started-at="2018-06-03T10:23:17Z" finished-at="2018-06-03T10:23:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_launch -->
        <test-method status="FAIL" signature="verify_login()[pri:0, instance:home.rainford.testexecution@604ed9f0]" name="verify_login" duration-ms="6442" started-at="2018-06-03T10:23:26Z" finished-at="2018-06-03T10:23:32Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at home.rainford.Admin.Branches_option(Admin.java:12)
	at home.rainford.Resprository.login(Resprository.java:20)
	at home.rainford.testexecution.verify_login(testexecution.java:11)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_login -->
      </class> <!-- home.rainford.testexecution -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
